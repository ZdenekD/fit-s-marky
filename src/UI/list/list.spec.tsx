import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import List from '.';

describe('List', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<List isAnimated={false} />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    })

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-list');

        expect(component.exists()).toBe(true);
    });
})
