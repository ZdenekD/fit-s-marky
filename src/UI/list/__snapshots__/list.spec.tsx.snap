// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List match snapshot 1`] = `
<ul class="list "
    data-test="component-list"
>
</ul>
`;
